hubmap_commons/hm_auth.py

```diff
@@ -14,30 +14,33 @@
 from hubmap_commons import file_helper
 from hubmap_commons import hubmap_const
 from hubmap_commons import exceptions
+from pickle import TRUE
```

Comment:
We are not using this import `from pickle import TRUE`, don't know why it was introduced.

Reply:
@yuanzhou Not sure how it got there...removing it now.